content/src/service/validations/Validations.ts

```diff
@@ -261,4 +262,84 @@ export class Validations {
       return avatar.avatar.snapshots[fileNameWithoutExtension] === hash
     })
   }
+
+  private static async calculateDeploymentSize(
+    deployment: DeploymentToValidate,
+    externalCalls: ExternalCalls
+  ): Promise<number> {
+    let totalSize = 0
+    for (const [, hash] of deployment.entity.content ?? []) {
+      const uploadedFile = deployment.files.get(hash)
+      if (uploadedFile) {
+        totalSize += uploadedFile.byteLength
+      } else {
+        const contentSize = await externalCalls.fetchContentFileSize(hash)
+        if (!contentSize) throw new Error(`Couldn't fetch content file with hash: ${hash}`)
+        totalSize += contentSize
+      }
+    }
+    return totalSize
+  }
+
+  /** Validate that given wearable deployment includes a thumbnail with valid format and size */
+  static readonly WEARABLE_CUSTOM: Validation = async (args) => {
```

Comment:
Let's leave this validation as the only one, and make `WEARABLE_THUMBNAIL` and `WEARABLE_SIZE` private methods. Also, let's change the name `WEARABLE_CUSTOM` to something more explicit, for example, `WEARABLE_FILES`

Reply:
will change the name!

other methods are not private since I added some tests for them and need to be visible 😞